src/service/resources-impl.js

```diff
@@ -239,43 +278,147 @@ export class ResourcesImpl {
     }
   }
 
+  /**
+   * @param {!Array<!IntersectionObserverEntry>} entries
+   * @param {!IntersectionObserver} unusedObserver
+   * @private
+   */
+  intersects_(entries, unusedObserver) {
+    dev().fine(TAG_, 'intersect', entries);
+
+    const toUnload = [];
+
+    const promises = entries.map(entry => {
+      const {boundingClientRect, isIntersecting, target, rootBounds} = entry;
```

Comment:
Why not just `entires.map( (({boundingClientRect, isIntersecting, ...}) => {...})`?

Reply:
It's because of Closure not checking destructures referencing banned properties. It's easier to fix in Babel when it's a variable declaration, much harder when its a param.

Reply:
Right. That was fixed in Closure I believe, btw.